backend/game/cards/discard_cards/victory_point_manipulation_cards/test_heal.py

```diff
@@ -0,0 +1,10 @@
+from game.cards.discard_cards.victory_point_manipulation_cards.heal import Heal
```

Comment:
These cards should not be under the victory_point_manipulation_cards folder...

There should be a health_manipulation_cards folder.  Personally not a fan of breaking it down this way (if a card does mutiple things, where does it go?) but for right now that is the pattern we are sticking with

Reply:
I think this was actually written by @otaiwo70 and I accidentally pulled it in. Tagged so he sees your comment.